src/index.html

```diff
@@ -9,6 +9,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="logo">
+          <div class="logo__img"> <a href="#home"><img src="images/logo.png" alt="moyo-logo"></a></div>
+        </div>
+        <div class="menu">
+          <nav class="menu__item menu__item1"><a href="#" class="menu__item__link is-active">Apple</a></nav>
```

Comment:
why do you need an additional class with the ordinal number of the menu item?
using numbers in a class name is bad practice
```suggestion
          <nav class="menu__item menu__item1">
            <a href="#apple" class="menu__item__link is-active">Apple</a>
          </nav>
```